src/main/certificateManager.ts

```diff
@@ -70,5 +70,6 @@ export class CertificateManager {
                 log.error(`There was a problem using the selected certificate: ${e}`);
             }
         }
+        this.certificateRequestCallbackMap.delete(server);
```

Comment:
Similar question here. Was this just not needed before since we'd only ever call this once per server before?

Reply:
Yeah I think this was just missed, I don't see any reason to keep the request in the callback map after the cert has been approved or the request was cancelled. If we wanted to re-initiate the request we would just reload the page.